src/content/developers/docs/bridges/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Bridges
+description: An overview of bridging for developers
+lang: en
+sidebar: true
+---
+
+With the proliferation of L1 blockchains and L2 [scaling](/developers/docs/scaling/) solutions, alongside an ever-growing number of decentralized applications going cross-chain, the need for communication and asset movement across chains has become an essential part of network infrastructure. Different types of bridges exist to help make this possible.
+
+## Need for bridges {#need-for-bridges}
+
+Bridges exist to connect blockchain networks. They enable connectivity and interoperability between blockchains.
+
+Blockchains exist in siloed environments, meaning there is no way for blockchains to trade and communicate with other blockchains naturally. As a result, while there could be significant activity and innovation within an ecosystem, it is limited by the lack of connectivity and interoperability with other ecosystems.
+
+Bridges offer a way for isolated blockchain environments to connect with each other. They establish a transportation route between blockchains where tokens, messages, arbitrary data, and even [smart contract](/developers/docs/smart-contracts/) calls can be transferred from one chain to another.
+
+## Benefits of bridges {#benefits-of-bridges}
+
+Put simply, bridges unlock numerous use cases by allowing blockchain networks to exchange data and move assets between them.
+
+Blockchains have unique strengths, weaknesses, and approaches to building applications (such as speed, throughput, costliness, etc.). Bridges help the development of the overall crypto ecosystem by enabling blockchains to leverage the innovations of each other.
+
+For developers, bridges enable the following:
+
+- the transfer of any data, information, and assets across chains.
+- unlocking new features and use cases for protocols as bridges expand the design space for what protocols can offer. For example, a protocol for yield farming originally deployed on Ethereum Mainnet can offer liquidity pools across all EVM-compatible chains.
+- the opportunity to leverage the strengths of different blockchains. For example, developers can benefit from the lower fees offered by the different L2 solutions by deploying their dapps across rollups, and sidechains and users can bridge across them.
+- collaboration among developers from various blockchain ecosystems to build new products.
+- attracting users and communities from various ecosystems to their dapps.
+
+## How do bridges work? {#how-do-bridges-work}
+
+While there are many [types of bridge designs](https://blog.li.fi/what-are-blockchain-bridges-and-how-can-we-classify-them-560dc6ec05fa), three ways to facilitate the cross-chain transfer of assets stand out:
+
+- **Lock and Mint –** Lock assets on the source chain and mint assets on the destination chain.
+- **Burn and Mint –** Burn assets on the source chain and mint assets on the destination chain.
```

Comment:
Is it worth mentioning the side of "Burn and unlock", ie.. You lock ETH, mint WETH somewhere else... but then to get it back you burn the WETH and _unlock_ your ETH, rather than _mint_ new ETH. 

I've also seen this summed up as "Lock-Mint-Burn"... Lock on chain 1, mint on chain 2, and when done, burn on chain 2 to unlock asset on chain 1.

Reply:
I suppose this would be a way to describe the full life cycle of an asset bridge. I think the idea here is to describe the different types of bridging and not the full life cycle of asset movement. Ill try and add an idea for some text here.